app/views/dashboard/_reports_widget.html.erb

```diff
@@ -14,13 +14,13 @@
     <tbody>
       <% events.each do |report| %>
         <tr>
-          <td><%= link_to trunc_with_tooltip(report.host, 14), host_config_reports_path(report.host) %></td>
-          <td><%= report_event_column(report.applied, "label-info") %></td>
-          <td><%= report_event_column(report.restarted, "label-info") %></td>
-          <td><%= report_event_column(report.failed, "label-danger") %></td>
-          <td><%= report_event_column(report.failed_restarts, "label-warning") %></td>
-          <td><%= report_event_column(report.skipped, "label-info") %></td>
-          <td><%= report_event_column(report.pending, "label-info") %></td>
+          <td class='ellipsis'><%= link_to report.host, host_config_reports_path(report.host) %></td>
```

Comment:
@lzap please try avoiding pulling another PR review into this one, if you do wish to get someone attention link it from the other pr.